src/server/api/routers/reviews.ts

```diff
@@ -51,6 +51,37 @@ export const reviewsRouter = createTRPCRouter({
         });
       }
     }),
+
+  referApplicant: protectedProcedure
+    .input(reviewSaveSchema)
```

Comment:
could we make a new schema for this input that just takes in a applicantUserId?

Reply:
Just to clarify, the schema should have reviewerUserId and applicantUserId?

Reply:
sorry I totally messed up the review comment. Just the applicantUserId, since the reviewerUserId can be found from the session (as is already implemented)

Reply:
gotcha made the changes, mb for delay (finally healthy again)